modules/ipi-install-extracting-the-openshift-installer.adoc

```diff
@@ -5,15 +5,23 @@
 [id="extracting-the-openshift-installer_{context}"]
 = Extracting the {product-title} installer
 
-After retrieving the installer, the next step is to extract it:
```

Comment:
These should probably just be numbered steps instead of having to say "First," "Then,", etc.